src/client/transport/local.ts

```diff
@@ -44,6 +45,8 @@ export function GetBotPlayer(state: State, bots: Record<PlayerID, any>) {
 interface LocalMasterOpts {
   game: Game;
   bots: Record<PlayerID, any>;
+  persist: boolean;
+  storageKey: string;
```

Comment:
Oh, one more thing. `bots`, `persist` and `storageKey` should all be optional here:

```ts
interface LocalMasterOpts {
  game: Game;
  bots?: Record<PlayerID, any>;
  persist?: boolean;
  storageKey?: string;
}
```